cgi-bin/LJ/Widget/NavStripChooser.pm

```diff
@@ -36,7 +36,7 @@ sub render_body {
     $ret .= "<p>" . $class->ml('widget.navstripchooser.colors') . "</p>";
 
     # choose colors
-    my $color_selected = $u->prop( 'control_strip_color' ) ne ''
+    my $color_selected = ( $u->prop( 'control_strip_color' ) // '' ) ne ''
```

Comment:
This bit of code seems unrelated to the original fix, yeah? If so, it should be pulled out into a separate pull request so we don't cross the streams.